### ISSUE TITLE
Style/HashSyntax is suggesting I use invalid syntax

### ISSUE THREAD

**Issue by agargiulo** [bbatsov/rubocop#1764]
``` bash
[12:41:30] % rubocop -V
0.30.0 (using Parser 2.2.0.3, running on ruby 2.1.5 x86_64-darwin14.0)

[12:42:36] % rubocop --display-cop-names spec/spec_helper.rb
Inspecting 1 file
C

Offenses:

spec/spec_helper.rb:55:7: C: Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
      :[] => 'abcdefghijklmnopqrstuvwxyzabcdefghijklmn'
      ^^^^^^

1 file inspected, 1 offense detected

[12:42:45] % rubocop --display-cop-names spec/spec_helper.rb
Inspecting 1 file
E

Offenses:

spec/spec_helper.rb:55:9: E: unexpected token tCOLON
      []: 'abcdefghijklmnopqrstuvwxyzabcdefghijklmn'
        ^
spec/spec_helper.rb:101:1: E: unexpected token $end

1 file inspected, 2 offenses detected
```

**Comment by agargiulo** [comment]
Appears that #1765 would fix this.

**Comment by bbatsov** [comment]
Fixed by #1765.
